aireview/pyproject.toml

```diff
@@ -9,11 +9,16 @@ packages = [{ include = "aireview", from = "src" }]
 [tool.poetry.dependencies]
 python = "^3.11"
 requests = "^2.32.3"
-pytz = "^2024.2"
 msrest = "^0.7.1"
 azure-devops = "^7.1.0b4"
 poetry-plugin-dotenv = "^2.4.0"
+prefect = "^3.1.5"
 
```

Comment:
**style**: Prefect dependency is listed twice - both in main dependencies and prefect group. Remove the duplicate entry from main dependencies if it's specific to the prefect functionality.